config/locales/en/providers.yml

```
@@ -250,6 +251,16 @@ en:
         singular:
           warning_text: The date you entered is over one month old.
           details_incorrect: No, I need to change this date
+
+    confirm_non_means_tested_applications:
+      show:
+        title:
+          under_18_now: No means test required as client is under 18
+          under_18_when_delegated_functions_used: |
+            You do no need to do a means test as your client was under 18
```

```suggestion
            You do not need to do a means test as your client was under 18
```
Unless we are expected to read it in a Scottish accent? :)